Trippy/Utils/DateUtils.h

```diff
@@ -9,11 +9,17 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef struct TimeInHrMin {
+    int hours;
+    int minutes;
+} TimeInHrMin;
+
 @interface DateUtils : NSObject
-+ (NSString *)formatDateAsISO8601:(NSDate *)date;
-+ (NSDate *)isoStringToDate:(NSString *)isoString;
++ (NSString *)formatDateAsIso8601:(NSDate *)date;
++ (NSDate *)iso8601StringToDate:(NSString *)isoString;
 + (int)aheadSecondsFrom1970:(NSDate *)date aheadBy:(int)aheadBy;
-+ (NSArray *)secondsToHourMin:(int)seconds;
++ (TimeInHrMin)secondsToHourMin:(int)seconds;
++ (NSNumber *) hourMinToSeconds:(TimeInHrMin)time;
```

Comment:
nit: consider just an `int` return here. `NSNumber` feels heavy (even if you're going to `NSNumber`-ify it later; can quickly do that with a `@(intVariable)`. That way it also has symmetry with your other fn above, which takes in an `int`.